CHANGELOG.md

```diff
@@ -58,6 +58,7 @@ END_UNRELEASED_TEMPLATE
   to the package path. This is enabled via the
   `# gazelle:experimental_allow_relative_imports` true directive ({gh-issue}`2203`).
 * (gazelle) Types for exposed members of `python.ParserOutput` are now all public.
+* (gazelle) Remove entry point file requirements when generating rules.
```

Comment:
Please reword to something along the lines of:

> `__init__.py` files no longer need to be present to generate a `BUILD.bazel` file in the directory.

As-is, I naive reader would not know that `__init__.py` is considered an entry point file, as "entry point" is typically parlance for the [`console_script` part of setuptools](https://packaging.python.org/en/latest/specifications/entry-points/).